### ISSUE TITLE
Making Multiple Controller Requests in Testing causes strange behaviour

### ISSUE THREAD

**Issue by see0** [flugger/laravel-responder#68]
Just want to inform other heavy laravel-responder users.

If you are making multiple calls within an unit test, eg:
```
function test_something{
  //one request
  $this->json('PUT');
  //second request
  $this->json('PUT');
}
```
You will experience strange behaviour. This is due to the fact - the responder binding is singleton. Hence some previously called responder configuration (such as with,without) will overlap with subsequent calls.

The culprit:
https://github.com/flugger/laravel-responder/blob/77523246e20ce1ac1faafe20d244aa0a7819df8e/src/ResponderServiceProvider.php#L238

I didn't have enough time to make the changes and run the test. And - this issue only applies in testing environment. 

As for workaround:

rebind the responder in your app service provider
```
    $this->app->bind('responder_test', function ($app) {
            return $app->make(Responder::class);
        });
```

create a new tests/bootstrap.php 
```
function responder()
{
    return app('responder_test');
}

include dirname(__FILE__) . "/../bootstrap/autoload.php";
```

change your phpunit.xml
```
         bootstrap="tests/bootstrap.php"
```

That's it. Hopefully, the author will put this into the readme.

**Comment by flugger** [comment]
Thanks for reporting! This is not the expected behavior, I’ll push a fix and tag a release later today :)

**Comment by flugger** [comment]
Hi again! This should now be fixed with version `2.0.11`.
